src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,99 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int MULTIPLICATION_FACTOR = 2;
+
+    private Node<K, V> next;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = (Node<K, V>[]) new Node[INITIAL_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        checkSize();
+        addNode(key, value);
+        size++;
     }
 
     @Override
     public V getValue(K key) {
+        int hash = hash(key);
+        int index = key == null ? 0 : indexOfMapArray(hash);
```

Comment:
you checked key == null  in hash method. I think, you can remove this line from your code and write something like 
int index = indexOfMapArray(key);